index.js

```diff
@@ -560,10 +560,13 @@ function buildObject (context, location) {
   let functionCode = `
   `
 
+  const nullable = schema.nullable === true
   functionCode += `
     // ${schemaRef}
     function ${functionName} (input) {
+      ${!nullable ? 'if (input === null) return \'{}\'' : ''}
       const obj = ${toJSON('input')}
```

Comment:
It should call the toJSON function first and then check for `null`.

Reply:
Should be ok now